data_model/src/events/pipeline.rs

```diff
@@ -61,7 +61,7 @@ impl EventFilter {
         self
     }
 
-    /// Filter by [`Hash`].
+    /// Filter by [`struct@Hash`].
```

Comment:
what does this mean?

Reply:
in `cargo doc` whenever you have an ambiguous name, e.g. both a trait and a struct with the same identifier, you can disambiguate the two using the @ symbol. 

If you run cargo doc locally, it will suggest you to do that. the alternative is fully qualifying the path, but IMO docs should almost never do that.